src/scmdata/dataframe.py

```diff
@@ -25,7 +25,8 @@
     years_match,
 )
 from .offsets import generate_range, to_offset
-from .pyam_compat import Axes, IamDataFrame, LongDatetimeIamDataFrame
+from .plotting import inject_plotting_methods
+from .pyam_compat import IamDataFrame, LongDatetimeIamDataFrame
```

Comment:
```suggestion
```

Reply:
`IamDataFrame` and `LongDatetimeIamDataFrame` are None if pyam isn't installed so they are used to check if pyam is installed

Reply:
smart